cartographer_ros/cartographer_ros/offline_node.cc

```diff
@@ -87,9 +111,13 @@ void RunOfflineNode(const MapBuilderFactory& map_builder_factory) {
   tf2_ros::Buffer tf_buffer;
 
   std::vector<geometry_msgs::TransformStamped> urdf_transforms;
-  if (!FLAGS_urdf_filename.empty()) {
-    urdf_transforms =
-        ReadStaticTransformsFromUrdf(FLAGS_urdf_filename, &tf_buffer);
+  for (const std::string& urdf_filename :
+       cartographer_ros::SplitString(FLAGS_urdf_filenames, ',')) {
+    const auto current_urdf_transforms =
+        ReadStaticTransformsFromUrdf(urdf_filename, &tf_buffer);
```

Comment:
Won't this have name clashes by default? I am unfamiliar with this part of the code, could you please explain the new behavior?

Reply:
`ReadStaticTransformsFromUrdf` reads static transforms from an urdf and dumps them in the tf buffer. For now, you can include robot names in link names in the different urdfs, so you don't get a clash. We can open an RFC for that if you would like to change this behaviour later.